lesson-3/src/main/java/lesson3/controllers/HelloController.java

```diff
@@ -17,4 +17,14 @@ public String getHello() {
 
     return "Hello!!!!";
   }
+
+  @GetMapping("/hoge")
+  public String getHogeHoge() {
+    String message = null;
+    if (true) {
+      System.out.printf("文字数" + message.length());
```

Comment:
\[FindBugs\] WARNING: NP: Null pointer dereference of message in lesson3\.controllers\.HelloController\.getHogeHoge\(\)